src/Types/Interval.purs

```diff
@@ -79,13 +79,14 @@ import Aeson.Decode ((</$\>), (</*\>))
 import Aeson.Decode as Decode
 import Aeson.Encode ((>$<), (>/\<))
 import Aeson.Encode as Encode
+import Aeson.Utils (maybeToEither)
```

Comment:
`note`?

Reply:
I don't know why `Aeson.Utils` re-implements `note` under a different name. I'd definitely prefer if we use `note`, we use it extensively elsewhere